CPUMeter.c

```diff
@@ -83,12 +83,12 @@ static void CPUMeter_display(Object* cast, RichString* out) {
       xSnprintf(buffer, sizeof(buffer), "%5.1f%% ", this->values[CPU_METER_SOFTIRQ]);
       RichString_append(out, CRT_colors[METER_TEXT], "si:");
       RichString_append(out, CRT_colors[CPU_SOFTIRQ], buffer);
-      if (this->values[CPU_METER_STEAL]) {
+      if (this->values[CPU_METER_STEAL] < 0.005) {
```

Comment:
This seems odd. Don't we want to display significant values?

Reply:
For this value I couldn't find documentation on how to distinguish no return from the system/kernel and just no CPU time spent on this. Suggestions on how to do this in a cleaner way welcome.

Reply:
https://github.com/htop-dev/htop/blob/5ea13e7ea9ca19130d5114a745d337b052cf2199/linux/Platform.c#L180 ?